tools/releaseBuild/azureDevOps/templates/release-MsixBundle.yml

```diff
@@ -0,0 +1,45 @@
+parameters:
+  jobName: ""
+  displayName: ""
+  imageName: ""
+  globalToolExeName: 'pwsh.exe'
+  globalToolPackageName: 'PowerShell.Windows.x64'
+
+
+jobs:
+- job: ${{ parameters.jobName }}
+  displayName: ${{ parameters.displayName }}
+  pool:
+    vmImage: ${{ parameters.imageName }}
+  steps:
+    - checkout: self
+      clean: true
+
+    - task: DownloadPipelineArtifact@2
+      inputs:
+        source: specific
+        project: PowerShellCore
+        pipeline: '696'
+        preferTriggeringPipeline: true
+        runVersion: latestFromBranch
+        runBranch: '$(Build.SourceBranch)'
+        artifact: finalResults
+        patterns: '**/*.msix'
+        path: '$(Pipeline.Workspace)/msix'
+
+    - pwsh: |
+        if ($null -eq Get-Command makeappx -ErrorAction Ignore) {
+          $msixUrl = '$(makeappUrl)'
+          Invoke-RestMethod -Uri $msixUrl -OutFile '\makeappx.zip'
+          Expand-Archive '\makeappx.zip' -destination '\' -Force
+        }
+      displayName: Install tools
+
+    - pwsh: |
+        $file = Get-ChildItem "$(Pipeline.Workspace)/msix" | Select-Object -First 1
+        $release = ($file.BaseName -split "-")[2]
+        $prefix = ($file.BaseName -split "-win")[0]
+        $pkgName = "$prefix.msixbundle"
+        Write-Verbose -Verbose "Creating $pkgName"
+        makeappx.exe bundle /d "$(Pipeline.Workspace)/msix" /p "$(Build.StagingDirectory)\signedPackages\$pkgName"
```

Comment:
This template is called from the release pipeline, which is triggered after the release build is done.
That means the `*.msixbundle` is generated after release build. However, `windows-package-signing.yml` is used in release build, and that means it will attempt to upload `*.msixbundle` during the release build. This seems not right.

Reply:
why?

Reply:
It looks to me the release build attempts to upload `.msibundle` but that file is not generated during release build. Let me know if I'm missing something.

Reply:
That's not technically a problem, there is a question if that's the way we want to do it.  I would say it should be like the rest of windows packaging and do this during the build.